documents/operating.md

```diff
@@ -0,0 +1,329 @@
+Although previously thought impossible, an ancient ritual found buried
+in the ruins of the Lawyer Cave describes the process of creating new
+BIFFUD Corporate Overlords.
+
+Any Corporate Overlord may nominate the addition of a new Overlord during
+a plotting session.  If the nomination is seconded, the ritual begins.
+
+1. The Corporate Overlords are presented with a biography of the candidate.
+2. The candidate submits a piece of original poetry or other creative work
+for the Overlords' enjoyment and consideration.
+3. A vote is held.
+
+If 70% or more of the Corporate Overlords approve the addition, then after the
+plotting session ends the new Corporate Overlord is onboarded and ownership of
+the company is redistributed so that each overlord owns an equal share.
+
+
+### Removing Corporate Overlords
+A Corporate Overlord can recuse itself from BIFFUD at any time by dying, uploading
+its consciousness to the cloud, or submitting a letter of resignation to both
+the President of BIFFUD and the President of the United States.
+
+Corporate Overlords can also vote to remove a peer from the organization if
+any of the following conditions apply:
+
+- The Corporate Overlord fails to attend 50% of the monthly plotting sessions
+in a given year without a note from a family member explaining its absence.
+- The Corporate Overlord has violated a serious federal law in the past year.
+- The Corporate Overlord is hated or despised.
+
+The removal of a corporate overlord can be nominated during a plotting session
+and requires 70% approval or more from the other Corporate Overlords.
+
+Overlords are entitled to a pro-rated profit share for their tenure in a given
+year, as well as ongoing profit-share according to their ownership of various
+BIFFUD projects.
+
+---------------------------------------------------------------------
+SECTION 3: 🤔🤔🤔 Projects 🤔🤔🤔
+---------------------------------------------------------------------
+- [x] Types of project
+- [x] Evaluating ideas
+- [x] IP (Intellectual Property / Internet Protocol)
+- [x] Open source
+
+### Types of Project
+A. **Grant Funded:** projects funded by grants from third parties are executed
+by the BIFFUD member who has led the relationship with the funders, or by an
+alternative BIFFUD member as delegated by that applicant.  Money associated with
+a grant is to be distributed as per the grant's agreement.
+
+B. **Member Owned:** projects that are led or adopted by BIFFUD members.  The
+owners of a project must all be BIFFUD members, and the percentages and specific
+terms of ownership should be define by those members as part of their application
+for the adoption of the idea under BIFFUD.
+
+C. **BIFFUD Core:** projects that are taken on by the entire corporation.  These
+might be projects utilized by the Corporate Overlords to execute on company policy,
+or simply be projects that the Corporate Overlords decide to develop without a concept
+of individual ownership.
+
+
+### Evaluating Ideas
```

Comment:
Great point, and you're right.  It is totally possible that "start to launch" could be a matter of hours.  It would be really good to be able to enable that type of turnaround, and that really would require a single person able to basically grant at least temporary acceptance.  I personally would still like that to be an *exception* and for it to also eventually culminate in some kind of more "official" acceptance through a more collective process.

Just to test run it using TING as an example:

1) TING Team could dive right in, on our own, throwing ideas and plans together, designing some hardware, figuring out potential costs, prototyping, maybe even iterating!

2) We would also submit an application (process TBD still, though probably involving a Pull Request)

3a) If the timeline demands expedited turnaround, Beastmaster could add a comment indicating a preliminary acceptance.

3b) Corporate Overlords could comment and vote over time -- maybe it gets enough votes to pass early -- feedback is provided via PR comments, PR can be updated.

3c) During the next in-sync plotting session we go through any project ideas that had not been resolved in the rolling method (basically, we treat the plotting session as a `flush()` on the applications that didn't get approved via rolling basis)

Does that process sound like it would address your concerns?